src/elements/ifc-client-link.ts

```diff
@@ -0,0 +1,30 @@
+import { Client } from '../client';
+const PATH_ATTR = 'path';
+
+/**
+ * A custom element responsible for rendering an anchor element, turning a path
+ * within the current application to a full top-level url.
+ * It is registered as `ifc-client-link` within client.ts
+ */
+
+export default class IfcClientLinkElement extends HTMLElement {
+  private _client: Client;
+
+  /** @internal */
+  constructor(clientInstance: any) {
+    super();
+    this._client = clientInstance;
+
+    const link = document.createElement('a');
+    link.setAttribute('target', '_top');
+    const slot = document.createElement('slot');
+    link.appendChild(slot);
+    const shadowRoot = this.attachShadow({ mode: 'open' });
+    shadowRoot.appendChild(link);
+    this._client.addListener('environmentalData', envData => {
+      let path = this.getAttribute(PATH_ATTR) || 'about:blank';
+      path = this._client.urlFromClientPath(path);
+      shadowRoot.firstElementChild?.setAttribute('href', path);
+    });
+  }
```

Comment:
I missed it before, but we'll want to add [attribute observation](https://developers.google.com/web/fundamentals/web-components/customelements#attrchanges) for the path attribute, and update the generated link accordingly when the attribute changes.  That should be able to use the same function as the environmentData listener, I think.